### PR TITLE
fix: querying wrt time

### LINKED ISSUE #179: Show all roles within time commitment range
## Steps to reproduce the problem

1. Apply the "time commitment" filter. For example, set the minimum time commitment to 9 hours and the maximum to 30 hours.

## Expected result after following these steps

The application displays any roles with a maximum time commitment above 9 hours and a minimum time commitment below 30 hours. So a role with a time commitment of 6-10 hours is also displayed.

## Actual result

The application displays any roles with a minimum time commitment over 9 hours and a maximum under 30 hours. So a role with a time commitment of 6-10 hours is NOT displayed. 

## Additional comments

I just realised that this only requires 2 lines to change in `src/GraphQL/roles.ts`:

```
          { timeCommitmentMin: { _gte: $timeCommitmentMin } }
          { timeCommitmentMax: { _lte: $timeCommitmentMax } }
```

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
### Issues resolved by these changes

Resolves #179

### CODE REVIEW

File: src/GraphQL/roles.ts [hunk 1]
```diff
@@ -56,8 +56,8 @@ export const RolesSearchQuery = gql`
         _and: [
           { localGroupId: { _in: $localGroupIds } }
           { workingCircleId: { _in: $workingCircleIds } }
-          { timeCommitmentMin: { _gte: $timeCommitmentMin } }
-          { timeCommitmentMax: { _lte: $timeCommitmentMax } }
+          { timeCommitmentMin: { _lte: $timeCommitmentMax } }
+          { timeCommitmentMax: { _gte: $timeCommitmentMin } }
```
Comment (SamLubbers) [inline src/GraphQL/roles.ts:60]:
```suggestion
          {
            _or: [
              { timeCommitmentMax: { _lte: $timeCommitmentMax } }
              { timeCommitmentMin: { _gte: $timeCommitmentMin } }
            ]
          }
```

**Review by SamLubbers — CHANGES_REQUESTED** [review]
Hey @ShubhamPalriwala congratulations with you first pull request :tada: The suggestion I had commented in the issue doesn't actually solve the problem. In your code I've suggested the code change that will do the trick :wink: 

I hope you can make it to the [meeting in a few hours](https://github.com/xrnl/aginun/blob/master/CONTRIBUTING.md#join-project-meetings) :crossed_fingers: Then we can meet each other and help you find interesting issues :call_me_hand:

**Comment by ShubhamPalriwala** [comment]
@SamLubbers Thanks for the help!

I'm really sorry but I have my end-semester exams this week so will not be able to join for the weekly meeting this time.

But would love to join for the next one and help fix some bugs and make the platform better

**Review by SamLubbers — CHANGES_REQUESTED** [review]
@ShubhamPalriwala you're almost there! Now you've added the new filter but not removed the old one. If you remove the old time commitment filter it should work :ok_hand: 

Did you test the changes locally? Then you would have probably seen that it doesn't work exactly as it should. Let me know if you need help with getting the application to run locally :pray: :seedling: 

And no worries about not joining the meeting! Good luck with your exam week :muscl

**Comment by ShubhamPalriwala** [comment]
@SamLubbers, I've made the requested changes and sorry for the number of 3 commits, It'd be better if you squashed and then merged it so that it's simple and easy to understand in the main branch.

**Review by SamLubbers — APPROVED** [review]
@ShubhamPalriwala no worries! We have commit squashing enabled by default when merging a pull request :sunglasses: 

I will now approve & merge. Congratulations with your first contribution to the project :tada: :partying_face:

**Pull request merged**